Library/Homebrew/gpg.rb

```diff
@@ -0,0 +1,47 @@
+require "utils"
+
+class Gpg
+  # Should ideally be using `GPGRequirement.new.gpg2`, etc to get path here but
+  # calling that directly leads to:
+  # requirement.rb:139:in `which_all': uninitialized constant Requirement::ORIGINAL_PATHS (NameError)
```

Comment:
You could/should probably do the reverse and have the `GPGRequirement` use the code from here. If that still doesn't work, shout and I'll try to get it working.